buildall.sh

```diff
@@ -16,7 +16,12 @@
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
 PIDS=()
-
+# some systems do not have enough tty's available to run all the
+# dumbinit tests in parallel. If the tty tests are failing try
+# setting SERIAL_BUILD=1
+if [ "$SERIAL_BUILD" == "" ]; then
+    SERIAL_BUILD=0
```

Comment:
I'm ok on all spaces, i had the wrong .emacs file on this machine.  We should pick an indentation level for shell scripts though. your's was 7 which is odd, and many were spaces.  I went to 4 on this forcepush